float64 GSegmSigma
int32 GSegmGrThrs
int32 GSegmMinSize
bool show
